src/extension/ios/iOSPlatform.ts

```diff
@@ -92,24 +92,24 @@ export class IOSPlatform extends GeneralMobilePlatform {
             },
         };
 
-        extProps = TelemetryHelper.addReactNativeVersionToEventProperties(this.runOptions.reactNativeVersion, extProps);
+        extProps = TelemetryHelper.addReactNativeVersionToEventProperties(this.runOptions.reactNativeVersions.reactNativeVersion, extProps);
```

Comment:
It seems that we get the version twice here.

Reply:
I'll remove redundant getting of version.